java/client/src/test/java/glide/api/models/TransactionTests.java

```diff
@@ -946,6 +948,9 @@ InfScoreBound.NEGATIVE_INFINITY, new ScoreBoundary(3, false), new Limit(1, 2)),
         transaction.functionDelete("LIB");
         results.add(Pair.of(FunctionDelete, buildArgs("LIB")));
 
+        transaction.copy("key1", "key2", true);
```

Comment:
test copy with DB index argument

Reply:
copy with DB does not work under cluster mode, so it was tested under `/standalone/CommandTests.java`, is it required here? `transaction` here is declared as a superclass object and doesn't pull up the copy() signature that is under `/api/models/Transaction.java`

Reply:
I see it is already added. I was confused that number of UTs is less than number of commands added to transaction classes.